### PR TITLE
DSWO-46 - Addressing various feedback points on the event page

### PR DESCRIPTION
**Pull request by helenb**
[Ticket URL](https://national-archives.atlassian.net/browse/DSWO-46)

## About these changes

Addresses the following feedback points from DWO-46:

-  If Venue type is set as ‘In person’ the FE should show the ‘Venue address’ in the Key details bar instead of the text ‘In person’. Note that we strip out the line breaks and links from the field which is rich-text, in order to display it on one line and wrap it in a link.
- If Venue type is set as ‘In person’ the ‘Venue address’ shown in the Key details bar should also link to ‘Venue website’. - the link is added if the venue type is 'in person', and the venue website field has been filled out.
- If Venue type is set as ‘Hybrid’ please can the FE read ‘In person and online’ instead of ‘Hybrid’ (in the Key details bar and Booking information tab) as it’s plainer English. This has been changed in the back-end so this is updated everywhere it is used, including in the admin interface when editing the page.
-  If Venue type is set as ‘In person’ the booking information tab should not show the text ‘In person’. It’s enough for it to show the address (which it currently does).
-  If Venue type is set as ‘In person’ or ‘Hybrid’, please can you add ‘Venue space name’ (only if this field has data) to appear here above ‘Venue address’.
- Please can you add the ‘Get directions’ link to the bottom of the ‘Location’ section as in the design, and make it link to ‘Venue website’- this is done if the website field has been filled out.
- Hardcoded header text ‘What you’ll see and do’ should read ‘About this [Event type, in lower case]’ eg ‘About this talk’.
- Hardcoded header text ‘Who can come’ should read ‘Who it’s for’.

Not covered in this MR:
- Please format date spans in the key details bar as per [the rules spec-ed in Confluence](https://national-archives.atlassian.net/wiki/spaces/DSWhatson/pages/142573589/Time+date+format+display+rules). This also needs changing on the event card, and might be best done with some back-end changes that supply an 'event_dates_display' field to the various templates.
- Please can you add the ‘View all sessions’ CTA (to show all event dates/times) to appear, and limit the number that appear on pageload to 6, if there are more than 6 event dates/times (as in the design)- this will require some custom JavaScript so might be better done in a separate pull request.
- Understand if this comes from styles you don’t control but if possible can you fix the line spacing of the headings, please? As discussed on the ticket the ideal solution here is to add the tna-heading classnames to rich text headings, which would be a back-end change across the site.

## How to check these changes

Review the list of changes above by testing an event page on your local build.

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- [x] Ensured that PR includes only commits relevant to the ticket
- [x] Waited for all CI jobs to pass before requesting a review
- [ ] Added/updated tests and documentation where relevant

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

File: templates/whatson/event_page.html [hunk 1]
```diff
@@ -43,7 +43,17 @@
                                 <i class="fa-solid fa-location-dot event-meta__icon"></i>
 
                                 <div class="event-meta__details">
-                                    <p>{{ page.get_venue_type_display }}</p>
+                                    {% if page.venue_type == "in_person" and page.venue_address %}
+                                        {% if page.venue_website %}
+                                            <a class="event-details-banner__link" href="{{ page.venue_website }}">
+                                        {% endif %}
+                                                {{ page.venue_address|process_address|richtext }}
```
Comment (ahosgood) [inline templates/whatson/event_page.html:50]: Do we have a link to a website where the link text is an address?

Comment (ahosgood) [inline templates/whatson/event_page.html:50]: Addresses should ideally be wrapped in an `<address>` element

Comment (helenb) [reply templates/whatson/event_page.html:50]: Yes according to the feedback from Ashley here: https://national-archives.atlassian.net/browse/DSWO-46?focusedCommentId=60823 " If Venue type is set as ‘In person’ the ‘Venue address’ shown in the Key details bar should also link to ‘Venue website’."

Comment (helenb) [reply templates/whatson/event_page.html]: done in https://github.com/nationalarchives/ds-wagtail/pull/1467/commits/bf60a0c879ca7d3bdb787285a791045be1c06a85

**Review by jamesbiggs — APPROVED** [review]
Approved on the BE side 😀

**Comment by helenb** [comment]
Thanks @jamesbiggs :) @ahosgood are you happy to approve?

**Comment by helenb** [comment]
@ahosgood I'm going to go ahead and merge as I've responded to your feedback and James has approved. Holler if you want any follow-up changes.

**Pull request merged**